apps/aechannel/src/aesc_utils.erl

```diff
@@ -151,6 +152,11 @@ unpack_payload(Tx) ->
         {_, _}                            -> {error, bad_offchain_state_type}
     end.
 
+is_offchain_tx_type(channel_offchain_tx) -> true;
+is_offchain_tx_type(ga_meta_tx         ) -> true;
```

Comment:
🤔 ?

Reply:
Based on function call, I see that more meaningful name would be `is_legal_payload_of_channel_close_solo_tx`. If so, [doc](https://github.com/aeternity/protocol/blame/8b2724d14d8b4878b6b2c74850ec3f7f7716aeb2/serializations.md#L603) indicates that this shall be false:
> The payload is a serialized signed channel off-chain transaction or it is empty.

Reply:
```suggestion
is_offchain_tx_type(ga_meta_tx         ) -> true; %% TODO: inspect the most inner transaction for being off-chain type
```

And I will address it in FSM's Generalized Accounts awareness PR